src/libc/stdio.rs

```diff
@@ -152,6 +155,28 @@ fn fclose(env: &mut Environment, file_ptr: MutPtr<FILE>) -> i32 {
     }
 }
 
+fn fsetpos(env: &mut Environment, file_ptr: MutPtr<FILE>, pos: ConstPtr<fpos_t>) -> i32 {
+    let FILE { fd } = env.mem.read(file_ptr);
+
+    let res = posix_io::lseek(env, fd, env.mem.read(pos), SEEK_SET);
+    assert_ne!(res, -1);
```

Comment:
Any reason you have an assert in these two functions? There is already logging in `lseek`. I think you can just return `if res == -1 { -1 } else { 0 }` in both of these.